config.json

```diff
@@ -625,6 +625,14 @@
         ],
         "difficulty": 3
       },
+      {
+        "slug": "square-root",
+        "name": "Square Root",
+        "uuid": "8960514f-17af-4009-b334-d6bfc4722c80",
+        "practices": [],
+        "prerequisites": [],
```

Comment:
Haha, we really don't have much that could fit here.
Your solution uses pattern matching, so we could add that, also `basics-2` I suppose.